src/Giveaway.js

```diff
@@ -179,6 +180,14 @@ class Giveaway extends EventEmitter {
         return this.options.lastChance || this.manager.options.default.lastChance;
     }
 
+    /**
+     * Pause options for this giveaway
+     * @type {GiveawayPauseOptions}
+     */
+     get pauseOptions() {
+        return this.options.pauseOptions
```

Comment:
```suggestion
        return this.options.pauseOptions || defaultPauseOptions;
```
So you do not need to set it in the start function = saves db space.